### DISCUSSION TITLE
Is it possible to send multipart/mixed request using REST Client Reactive or REST Client?

### DISCUSSION THREAD

**Discussion by Ballsigno** [quarkusio/quarkus#28021] [Q&A] [answered]
Hi.
I did a bit of research about sending multipart/mixed request using REST Client Reactive or Rest Client, but I couldn't make it.
I tried at first using REST Client Reactive and soon understood `org.jboss.resteasy.reactive.MultipartForm` is only for `form-data`, and `resteasy-multipart-provider` is not for this.
So I checked the old REST Client, but the result is the same. I saw [this page](https://docs.jboss.org/resteasy/docs/1.1.GA/userguide/html/Multipart.html) and prepared MultipartOutput manually(like 21.6. Output with multipart), but it doesn't work when I use with `multipart/mixed`(But form-data+MultipartFormDataOutput works!).  Am I missing something or it's just not supported?

sample project: https://github.com/Ballsigno/quarkus-multipart-mixed-test
you can see a test class in both projects.

[a comment by an automated account is not shown]

**Comment by geoand** [comment, marked as answer]
It's not currently possible. Please open a new issue requesting this feature

**Reply by Ballsigno** [reply]
Thank you for your quick response. I opened an issue for this!

**Reply by Ballsigno** [reply]
Link: https://github.com/quarkusio/quarkus/issues/28026

**Reply by geoand** [reply]
Thanks
